src/test/java/mil/dds/anet/test/resources/RollupGraphComparatorTest.java

```diff
@@ -0,0 +1,71 @@
+package mil.dds.anet.test.resources;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.Assert;
```

Comment:
Please use this type of assertion:
```
import static org.assertj.core.api.Assertions.assertThat;
```
(see other test classes).

Reply:
Personally I do not like static imports, but have changed to be consistent with the rest of the tests.